docs/content/tutorial/step_09.ngdoc

```diff
@@ -38,16 +38,16 @@ Since we are using [Bower][bower] to install client-side dependencies, this step
   "license": "MIT",
   "private": true,
   "dependencies": {
-    "angular": "1.5.x",
-    "angular-mocks": "1.5.x",
-    "angular-route": "1.5.x",
+    "angular": "1.6.x",
+    "angular-mocks": "1.6.x",
+    "angular-route": "1.6.x",
```

Comment:
OK, I'll revert them here.

Do you think I can perhaps leave jQuery 3.1 -> 3.2 update? That shouldn't require any changes.